src/ServiceContainer/ServiceProvider/ServiceContainerServiceProvider.php

```diff
@@ -150,7 +150,34 @@ public function getContainerDefinition() {
       'arguments' => array('cron'),
     );
 
-    // @todo Make it  possible to register all ctools plugins here.
+    if (module_exists('ctools')) {
```

Comment:
should be

$this->moduleExists('ctools')

can't unfortunately use the moduleHandler here not yet.